### ISSUE TITLE
Are there pre-trained models, inference and training codes that are adapted for qwen-image-edit-2509?

### ISSUE THREAD

**Issue by cuililucy-dot** [CSU-JPG/Glance#7]
Hello, I noticed that your pre-trained models, inference and training codes are all based on qwen-image. However, the data processing module is adapted for qwen-image-edit-2509. I would like to ask if there are pre-trained models, inference and training codes that are adapted for qwen-image-edit-2509?

**Comment by zhuobaidong** [comment]
We actually tried Qwen-Image-Edit model and results looks great, it seems that Glance could extend to image edit and video generation task. we have not yet packaged the code and models related to Qwen-Image-Edit, more details can be found in the **_appendix_** of the Glance paper.

**Comment by minushuang** [comment]
When training Qwen Image Edit, with one sample and 100 steps, the slow lora loss was 0.003 after training, but the fast lora loss was 0.2. Is this normal?

**Comment by zhuobaidong** [comment]
It's difficult to judge, can i see more samples?
